.github/workflows/pr.yml

```diff
@@ -38,13 +44,8 @@ jobs:
         timeout-minutes: 20
         strategy:
             matrix:
-                node: [lts/*]
-                os: [windows-latest, macos-latest]
-                include:
-                    - os: ubuntu-latest
-                      node: lts/-1
-                    - os: ubuntu-latest
-                      node: latest
+                node: [14, 16, 18, 19, 20]
```

Comment:
Our goal is to support the last two `lts` versions alongside `latest`, so not sure of the motivation for this.

Reply:
Making two changes here:

1. Running our existing versions (16, 18, 20) on each system, both to ensure system specific problems are tested on each version, and to make it easier to troubleshoot whether a failure is related to the version or system
2. Supporting all the other [actively supported versions of Node](https://github.com/nodejs/Release) (14, 19)

If you don't want either of these, feel free to discuss

Reply:
I'd favor our current setup because it automatically stays up to date as new versions are released. I'm comfortable just saying "we support latest + last two LTS." If someone finds a bug on 19 that's fine, they can log it, but it seems very remote that it would not be caught by the tests for 18 or 20.

Reply:
I'm sticking with both your recommendation and my first change.